.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 {
   "deno.config": "./deno.json",
+  "deno.unstable": true,
```

Comment:
We are currently not using anything unstable right?

Reply:
^